src/layouts/index.tsx

```diff
@@ -0,0 +1,21 @@
+import Header from '../components/header';
+import NavBar from '../components/navBar';
+import { Outlet } from 'react-router-dom';
+import { Container, ContentWrapper, GlobalStyle, NavBarWrapper } from './styled';
+
+const Layout: React.FC = () => {
+  return (
+    <Container>
+      <NavBarWrapper>
+        <NavBar />
+      </NavBarWrapper>
+      <GlobalStyle />
+      <ContentWrapper>
+        <Header />
+        <Outlet />
```

Comment:
Keep in mind that a main content sjould be wrapped in a `main` tag

Reply:
fix: ContentWrapper changed from div to main

Reply:
You don't need to wrap Header and Nav Bar in `main` please read any doc about that